pilot/pkg/proxy/envoy/v1/externalservice.go

```diff
@@ -180,6 +183,16 @@ func buildExternalServiceVirtualHost(externalService *routingv2.ExternalService,
 	// FIXME: similarly, routing rules for other services that route to this external service will be constructed incorrectly
 	routes := buildDestinationHTTPRoutes(sidecar, service, port, instances, config, buildClusterFunc)
 
+	// inject Mixer calls per route.
+	// every route here belongs to the same destination.service, ie serviceName
+	// And source is the sidecar All attributes are directly sent to Mixer so none are forwarded.
+	if mesh.MixerAddress != "" && !mesh.DisablePolicyChecks {
```

Comment:
This disables egress reporting when policy checks are disabled. Is that desired? cc @douglas-reid

Reply:
Fixed it here and elsewhere.